source/billing.txt

```diff
@@ -227,6 +227,23 @@ of inactivity.
    - How many monthly active users does your app have?
    - How much time is an average user online each month?
 
+.. _billing-edge-server:
```

Comment:
Was going to ask if this should be added to the bullet list of usage types in the Overview section, but probably not as long as it's being billed with Device Sync